scripts/linters/python_linter.py

```diff
@@ -438,13 +481,18 @@ def _lint_py_files_for_python3_compatibility(self):
                 # This line invokes Pylint and prints its output
                 # to the target stdout.
                 python_utils.PRINT('Messages for Python 3 support:')
+                pylint_output = WritableObject()
```

Comment:
Follow all the comments I've left above for this case!

Reply:
Done.